frontend/src/components/QuestionInput/ErrorAlert.test.tsx

```diff
@@ -32,15 +34,19 @@ describe("Test the ErrorAlert component", () => {
   });
 });
 
-describe("Test the onClose function", () => {
-  it("calls the onClose function with the file upload preview ID when the close button is clicked", async () => {
-    const mockOnClose = jest.fn();
+describe("Test the onRemove function", () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
```

Comment:
Dust/hot tip: so fresh, so clean
```suggestion
  afterEach(jest.clearAllMocks);
```